README.md

```diff
@@ -56,7 +57,13 @@ interface ImageSource {
 }
 ```
 
-TODO: clarify that i mean Strong cannot have an ancestor of Strong etc
+### Teaser
+
+```ts
+import { TeaserProps as Teaser } from '@financial-times/x-teaser'
```

Comment:
Does this make sense as an approach? The alternative is copy/paste all the type definitions from x-teaser (~20 lines or so)

Reply:
Ooh yeah good point about versioning! That makes sense.

I'm only copying the types we need/use for recommended link teasers for now. I think at some point _someone_ should do a major version of x-teaser with a better data model (things like their image definition, and metaLinks are a bit weird). It requires thinking about all the non-article use cases though, so lets not couple ourselves to that.

Reply:
> Ooh yeah good point about versioning! That makes sense.

forever i think i'll be profoundly affected by the trouble faced by jake for polyfill.io in the javascript spec's definition of whitespace as "the latest unicode spec's definition of whitespace"

> lets not couple ourselves to that.

i'm sure we'll get there eventually ❤️